src/lib/_datatable-theme.scss

```diff
@@ -1,5 +1,4 @@
-@import '~@angular/material/core/theming/all-theme';
```

Comment:
That was a change in Material 2 beta.3, which is aligned for working with Angular 4.x. I've just merge a version 1.3.0 that will include that change and work with these latest versions.